compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java

```diff
@@ -614,13 +614,6 @@ private void setInvokableTypeSymbol(BInvokableType invokableType) {
         }
         BInvokableTypeSymbol tsymbol = (BInvokableTypeSymbol) invokableType.tsymbol;
         List<BVarSymbol> params = new ArrayList<>(invokableType.paramTypes.size());
-        for (BType paramType : invokableType.paramTypes) {
-            BVarSymbol varSymbol = new BVarSymbol(paramType.flags, Names.EMPTY, //TODO: should be written/read to BIR
-                                                  this.env.pkgSymbol.pkgID, paramType, null, symTable.builtinPos,
-                                                  COMPILED_SOURCE);
```

Comment:
Did we verify all of the fields in the existing param symbols are the same? Package ID, origin, etc.? Shall we add a test also?